bitnami/external-dns/README.md

```diff
@@ -285,6 +288,7 @@ $ helm install my-release \
   bitnami/external-dns
 ```
 
+
```

Comment:
Is it an extra line? if so, please remove it.

Reply:
done